### PR TITLE
Dont show empty fields in the contact and company detail view

### PR DESCRIPTION
**Pull request by volha-pivavarchyk**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [no]
| New feature/enhancement? (use the a.x branch)      | [yes]
| Deprecations?                          | [no]
| BC breaks? (use the c.x branch)        | [no]
| Automated tests included?              | [no] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

If we view detailed information about a contact or company on the view page, all fields are displayed. We propose to display only filled fields in this PR

Before:
image

After:
image

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new contact/company or open an existing one
3. Open details information
4. Only filled fields here

<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11016"></a>

### CODE REVIEW

**Review by kuzmany — CHANGES_REQUESTED** [review]
If value is 0 then field is not int he list. Usually boolean field, or select field with 0 value

**Comment by volha-pivavarchyk** [comment]
@kuzmany I agree with you. I was confused by the "Points" field, which defaults to 0.

File: app/bundles/LeadBundle/Tests/Controller/LeadDetailFunctionalTest.php [hunk 1]
```diff
@@ -42,13 +46,48 @@ public function testCustomFieldOrderIsRespected(): void
...
+                ],
+            ],
+        ]);
+        $leadFields = array_filter(
+            $lead->getFields(true),
+            function ($value) {
+                return isset($value['value']);
+            },
+            ARRAY_FILTER_USE_BOTH
+        );
```
Comment (escopecz) [inline app/bundles/LeadBundle/Tests/Controller/LeadDetailFunctionalTest.php]: I'm wondering why `ARRAY_FILTER_USE_BOTH` flag was used as the key is not used in the callback. Also, we can use arrow functions now 🚀 
```suggestion
        $leadFields = array_filter($lead->getFields(true), fn ($value) => isset($value['value']));
```

**Review by escopecz — DISMISSED** [review]
Looks good. I noticed one possible improvement, but since it's in the test and it does not change the functionality I'm approving it with or without the change.

**Review by escopecz — APPROVED** [review]
Re-approving after the latest commit 👍

**Review by kuzmany — APPROVED** [review]
Works as expected

**Pull request merged**